### PR TITLE
Add Debian 11 support

### PR DESCRIPTION
**Pull request by root-expert**

### CODE REVIEW

**Comment by root-expert** [comment]
No Debian 11 packages from Zabbix just yet

**Review by bastelfreak — APPROVED** [review]

**Comment by root-expert** [comment]
Waiting for Zabbix 4.0 packages which is still supported.

File: manifests/repo.pp [hunk 1]
```diff
@@ -134,10 +134,17 @@
             id     => 'A1848F5352D022B9471D83D0082AB56BA14FE591',
             source => 'https://repo.zabbix.com/zabbix-official-repo.key',
           }
+
+          # Debian 11 provides Zabbix 5.0 by default. This can cause problems for 4.0 versions
+          $pinpriority = $facts['os']['release']['major'] {
+            '11'    => 1000,
+            default => undef,
+          }
```
Comment (root-expert) [inline manifests/repo.pp:142]: Maybe I should pin it higher for all Debian? Since we are managing the repo we want to overrule system's default repos

Comment (root-expert) [reply manifests/repo.pp:142]: Some reference -> https://support.zabbix.com/browse/ZBX-19957?focusedCommentId=580023&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-580023

**Review by bastelfreak — APPROVED** [review]

**Pull request merged**